arm/Microsoft.Network/virtualNetworkGateways/.parameters/expressRoute.parameters.json

```
@@ -3,16 +3,16 @@
     "contentVersion": "1.0.0.0",
     "parameters": {
         "name": {
-            "value": "vnet-validate-01-ergw-01"
+            "value": "sxx-az-gw-er-001"
         },
         "gatewayPipName": {
             "value": [
-                "vnet-validate-01-ergw-01-pip-s-02"
+                "vsxx-az-gw-er-pip-001"
             ]
         },
         "domainNameLabel": {
             "value": [
-                "vnet-validate-01-ergw-01-pip-s-02"
+                "vsxx-az-gw-er-dm-001"
```

Is this a dependency or a deployed resource? If deployed
```suggestion
                "sxx-az-gw-er-dm-001"
```